### PR TITLE
Basic spring boot practice  

### PR DESCRIPTION
**Pull request by Shevadesuyash**
delete all branch and create again for this merge

### CODE REVIEW

**Comment by spreezy-tech** [comment]
@Shevadesuyash - Rename the title of the PR with appropriate title of the task

File: src/main/java/com/training/test/controller/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin(origins="*")
+public class AuthenticationController {
+
+    //    @Autowired
```
Comment (spreezy-tech) [inline src/main/java/com/training/test/controller/AuthenticationController.java]: Remove commented code. it is bad practice to keep unused commented code.

**Review by spreezy-tech — CHANGES_REQUESTED** [review]
@Shevadesuyash  -I see lot of commented code. overall your work looks very clean.  please remoce the commented code if not used

Great job Suyash.

File: src/main/java/com/training/test/controller/AuthenticationController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.training.test.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin(origins="*")
+public class AuthenticationController {
+
```
Comment (Shevadesuyash) [inline src/main/java/com/training/test/controller/AuthenticationController.java:27]: removed comments

**Comment by Shevadesuyash** [comment]
@spreezy-tech   Code is updated. Removed all unnecessary comments. 
Renamed PR .

**Comment by Shevadesuyash** [comment]
working code. Deploy on frontend team by creating image of project and run on frontend team.
@spreezy-tech  please check and validate PR  request.

**Review by spreezy-tech — APPROVED** [review]
Great work Suyash. Keep learning. Many more things to explore!

Approved

**Pull request merged**